ci/prs.yml

```diff
@@ -141,6 +141,7 @@ jobs:
       var_name: bash-lib
   - bash: |
       set -euo pipefail
+      cd sdk
```

Comment:
Why is this in this PR?

Reply:
Missing from a different PR, needed to make full compat work post moving into `sdk`

Reply:
It was specifically fixed in this PR
https://github.com/digital-asset/daml/commit/89a57673b07ef2d7f7a4d400eb5758425c2170f0
Which was a 2.x specific fix for the compatibility tests.
Given 3.x doesn't run the same tests, this wasn't forward ported